### PR TITLE
thiziribouaziz animation for Hachtoberfest

### PR DESCRIPTION
**Pull request by thiziribouaziz**
Thank you for making a pull request to ZTM's Animation Nation project. We will review your pull request as soon as we can, but keep in mind we can get very busy during Hacktoberfest.

While you're waiting, please look over this checklist. Feel free to write a description of what you did as well!

## What I did

*Optional: Edit this to tell us what you did!*

## Pull request checklist

- [x] This pull request uses **HTML and CSS** only and provides a `meta.json` file.
- [x] All files in this pull request are in a directory named: `<github_username>-<art_name>` without the <>.
- [x] Your pull request contains `index.html` and `styles.css`.
	- [x] Your html file is named exactly: `index.html`
	- [x] Your css file is named exactly: `styles.css`
- [x] Your pull request your `meta.json` with:
	- [x] `githubHandle`: being your unique github user name,
	- [x] `artName` : name of your animation
- [x] Your submission includes **at least one animation**.
- [x] Your pull request **does not modify any other files** in the repository.

#### Not normally needed
- [x] Check this if you would like to discuss an improvement besides adding animations

**Note: if your pull request makes an improvement besides adding a CSS animation you will need to discuss this with the maintainers. You can help kickstart this process by explaining what your change does and why you are making it in the "What I did" section**

## What to expect

- If your pull request meets the checklist requirements it will be approved and merged by a maintainer
- If your pull request needs some changes. A maintainer will leave a comment requesting changes
- If your pull request significantly violates the criteria in the checklist it will be closed with feedback.

### CODE REVIEW

File: Art/thiziri_bouaziz/index.html
Comment (LaurelineP) [inline Art/thiziri_bouaziz/index.html:1]: # Unaccepted SVG
Like mentioned in the readme, we do not accept SVG
Try to find another way to do an animation without svg

File: Art/thiziri_bouaziz/styles.css [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    background: #b22cff;
+  }
+}
+
+h1{
+    padding: 30px;
+    border-radius: 40px;
+    background-color: rgba(211, 10, 167, 0);
+    color: rgb(79, 12, 100);
+    /* position: absolute; */
```
Comment (LaurelineP) [inline Art/thiziri_bouaziz/styles.css:47]: ## Unnecessary comment
Please remove the commented line

**Review by LaurelineP — CHANGES_REQUESTED** [review]
Aloha :) And thanks for your contribution.
Before we could merge your MR: 
- [ ] Please address the feedbacks in your PR
- [ ] Please address the following: your foldername starting with your github username ( = handle ) is incorrect with the hyphen
( not present in your github username / github handle )
Please adjust accordingly

File: Art/thiziri_bouaziz/index.html
Comment (thiziribouaziz) [reply Art/thiziri_bouaziz/index.html:1]: yes i deleted it

File: Art/thiziri_bouaziz/styles.css (hunk 1 above)
Comment (thiziribouaziz) [reply Art/thiziri_bouaziz/styles.css:47]: yes i remove it

File: Art/thiziri_bouaziz/index.html
Comment (LaurelineP) [reply Art/thiziri_bouaziz/index.html:1]: @thiziribouaziz 
This svg is still in the DOM
image

**Review by LaurelineP — COMMENTED** [review]
Please review the new feedback

**Pull request closed without merging**